tests/models/test_jamba.py

```diff
@@ -21,17 +22,152 @@ def test_models(
 
     with hf_runner(model, dtype=dtype) as hf_model:
         hf_outputs = hf_model.generate_greedy(example_prompts, max_tokens)
+        hf_logprobs_outputs = hf_model.generate_greedy_logprobs_limit(
+            example_prompts, max_tokens, num_logprobs=2)
 
     with vllm_runner(model, dtype=dtype) as vllm_model:
         vllm_outputs = vllm_model.generate_greedy(example_prompts, max_tokens)
+        vllm_logprobs_outputs = vllm_model.generate_greedy_logprobs(
+            example_prompts, max_tokens, num_logprobs=2)
 
     for i in range(len(example_prompts)):
-        hf_output_ids, hf_output_str = hf_outputs[i]
-        vllm_output_ids, vllm_output_str = vllm_outputs[i]
-        assert hf_output_str == vllm_output_str, (
-            f"Test{i}:\nHF: {hf_output_str!r}\nvLLM: {vllm_output_str!r}")
-        assert hf_output_ids == vllm_output_ids, (
-            f"Test{i}:\nHF: {hf_output_ids}\nvLLM: {vllm_output_ids}")
+        _, hf_output_str = hf_outputs[i]
+        hf_output_ids, _, hf_output_logprobs = hf_logprobs_outputs[i]
+
+        _, vllm_output_str = vllm_outputs[i]
+        vllm_output_ids, _, vllm_output_logprobs = vllm_logprobs_outputs[i]
+
+        if hf_output_str != vllm_output_str:
```

Comment:
You might want to consider using the utilities under `tests/models/utils.py`.

Reply:
Thanks! Addressed, Regular tokens comparison is working so I switched to use those